Makefile.am

```diff
@@ -6584,6 +6586,11 @@ SYSINIT_TARGET_WANTS += \
 	systemd-sysctl.service \
 	systemd-ask-password-console.path
 
+if ENABLE_MACHINED
+MACHINES_TARGET_WANTS += \
+	var-lib-machines.automount
+endif
```

Comment:
this doesn't look right. nspawn uses the dir too (in fact, it's the dir's primary user if you so will), hence this should be pulled in if nspawn is enabled too.